testkit/src/main/java/org/apache/calcite/test/catalog/MockCatalogReaderSimple.java

```diff
@@ -99,6 +108,12 @@ private void registerTableEmpNullables(MockTable empNullablesTable, Fixture fixt
     empNullablesTable.addColumn("COMM", fixture.intTypeNull);
     empNullablesTable.addColumn("DEPTNO", fixture.intTypeNull);
     empNullablesTable.addColumn("SLACKER", fixture.booleanTypeNull);
+    empNullablesTable.setReferentialConstraints(
+        Lists.newArrayList(
+            RelReferentialConstraintImpl.of(
+        Lists.newArrayList(DEFAULT_CATALOG, DEFAULT_SCHEMA, "EMPNULLABLES"),
+        Lists.newArrayList(DEFAULT_CATALOG, DEFAULT_SCHEMA, "DEPT"),
+        Lists.newArrayList(IntPair.of(7, 0)))));
```

Comment:
Same here, a comment mentioning the column name from both tables would help clarify

Reply:
Yeah, add it.